src/barkha/Assignment1.java

```diff
@@ -0,0 +1,96 @@
+
+3) Reset the form, for that click on �Reset form in same tab� button.
+
+4) Verify Reset works properly on your text field, radio button and checkbox. (For each type, min one validation is required.)
+
+5) Click on the �Go and Practice for it� Button.
+
+6) Website redirect to �automationbykrishna.com�.
+
+7) Verify title and URL of redirected page website.
+    Title should be �Login Signup Demo� and the URL should be �http://automationbykrishna.com/�.
+*/
+package barkha;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Assignment1 {
+	
+	public static void main(String[] args) {
+		System.setProperty("webdriver.chrome.driver", "./resources/windows/chromedriver.exe");
+		WebDriver driver=new ChromeDriver();
+		driver.get("file:///C:/Users/DELL%201801/Downloads/Microsoft.SkypeApp_kzf8qxf38zg5c!App/All/SeleniumAssignment_1%20(3).html");
+		
+		//verify correct title
+		if (driver.getTitle().equals("TECHNOCREDITS"))
+			System.out.println("You are on a correct Page having title TECHNOCREDITS.");
+		else
+			System.out.println("You are on a incorrect page.");
+		
+		//filling all details in the form
+		WebElement firstName=driver.findElement(By.id("first name"));
+		firstName.sendKeys("Barkha");
+		
+		WebElement lastName=driver.findElement(By.id("last name"));
+		lastName.sendKeys("Patle");
+		
+		WebElement email=driver.findElement(By.id("E-mail"));
+		email.sendKeys("barkhapatle01@gmail.com");
+		
+		WebElement company=driver.findElement(By.id("Company Name"));
+		company.sendKeys("Syntel");
+		
+		//if female radio button is selected then click on male and vice versa
+		if (driver.findElement(By.id("femaleG"))!=null) {
+			WebElement maleButton= driver.findElement(By.id("maleG"));
+			maleButton.click();
+		}else {
+			WebElement femaleButton=driver.findElement(By.id("femaleG"));
+			femaleButton.click();
+		}
+		
+		WebElement exp=driver.findElement(By.id("entry2"));
+		exp.click();
+		
+		WebElement lang=driver.findElement(By.id("knownlanguages"));
+		lang.sendKeys("Java");
+		
+		WebElement learnLang1=driver.findElement(By.id("java"));
+		learnLang1.click();
+		
+		WebElement learnLang2=driver.findElement(By.id("python"));
+		learnLang2.click();
+		
+		WebElement comName=driver.findElement(By.id("google"));
+		comName.click();
+		
+		WebElement terms=driver.findElement(By.id("termsAndConditions"));
+		terms.click();
+		
+		//click on reset button
+		driver.findElement(By.id("resetBtn")).click();
+		
+		//checking reset button is working or not
+		if(firstName.isSelected()==false && lastName.isSelected()==false &&  email.isSelected()==false && company.isSelected()==false && exp.isSelected()==false && lang.isSelected()==false && learnLang1.isSelected()==false && learnLang2.isSelected()==false && comName.isSelected()==false && terms.isSelected()==false) {
```

Comment:
Where is text field validation ?

Reply:
Hi Krishna, I provided validation of reset button on all fields(text boxes/check boxes /radio button) in the single line with && operator here. Plz correct if I am wrong here. e.g.,(firstName.isSelected()==false) is for text field.

Reply:
That's what I am saying isSelected() you should use on radio button and checkbox not on the text field. 
For textfield you should use driver.findElement(By.id("first name")).getAttribute("value").equals("") like this.

Reply:
yes,... changes done